crates/ark/src/reticulate.rs

```diff
@@ -89,11 +98,13 @@ pub unsafe extern "C" fn ps_reticulate_open(input: SEXP) -> Result<SEXP, anyhow:
     let main = RMain::get();
 
     let input: RObject = input.try_into()?;
-    let input_code: Option<String> = input.try_into()?;
-
-    let mut comm_id_guard = RETICULATE_COMM_ID.lock().unwrap();
+    let input_code: Option<String> = match r_is_null(input.sexp) {
```

Comment:
This is not the focus of this PR, but would be nice to sneak it here because reticulate by default will send a `NULL`.

Reply:
Add a comment about this reticulate behaviour?